cmd/autotestgridgenerator/main.go

```diff
@@ -98,9 +99,26 @@ func main() {
 		logrus.Info("no new commits, exiting ...")
 		os.Exit(0)
 	}
+	newRepo := githubRepo
+	gc.SetMax404Retries(0)
+	if _, err := gc.GetRepo(o.githubLogin, githubRepo); err != nil {
+		// Somehow github.IsNotFound doesn't recognize this?
```

Comment:
This looks suspicious, have you looked at why it doesn't? We should fix that instead of relying on `strings.Contains` here.